docs/source/contributing/testing.rst

```diff
@@ -1,21 +1,24 @@
 ============
 Adding Tests
 ============
-When adding a new feature, it should always be tested. Tests prevent
-manim from breaking at each new feature added by checking if any other
+If you are adding new features to manim, you should add appropriate tests for them. Tests prevent
+manim from breaking at each change by checking that no other
 feature has been broken and/or been unintentionally modified.
```

Comment:
> When add new features to manim, they should be tested appropriately, to ensure that no other parts of the codebase has been broken and/or unintentionally modified as a result of the change.

Perhaps this may sound better?

Reply:
I prefer my wording as I don't think the "they" referring to the features is grammatically correct, and rewriting it would still feel run-on.